packages/core/src/base-chart.ts

```diff
@@ -238,6 +238,22 @@ export class BaseChart {
 		}
 	}
 
+	getFillColor(datasetLabel: any, label?: any, value?: any) {
+		if (this.options.getFillColor && !this.options.accessibility) {
```

Comment:
what if accessibility is on?

Reply:
Then this feature isn't available - using color to signify data is not accessible. We'll need a way to generate a unique pattern for each custom color.